src/loadMessages.js

```diff
@@ -0,0 +1,11 @@
+export default function loadMessages(lang) {
+  const wrapped = import(`./messages/${lang}`)
+
+  if (typeof window !== 'undefined') {
+    return wrapped.then(msgs => msgs.default)
+  }
+
+  /* eslint-disable global-require, import/no-dynamic-require, prefer-template */
+  // return Promise.resolve(require('./messages/' + lang).default)
+  return __webpack_require__(wrapped.resolve()).default
+}
```

Comment:
Here's how I verified. Look how `chunkName()` is the same as the `resolve()` function you're already grabbing off the *universal config* object:

![screen shot 2017-08-13 at 4 32 06 am](https://user-images.githubusercontent.com/154732/29249232-7b85d346-7fe0-11e7-882b-bc4989707324.png)